modules/apps/map/map-common/src/main/resources/META-INF/resources/js/MapBase.js

```diff
@@ -81,15 +81,15 @@ class MapBase extends EventEmitter {
 	 * @param  {Array} args List of arguments to be sent to State constructor
 	 * @review
 	 */
-	constructor(args) {
+	constructor(args = {}) {
 		super(args);
 
 		const {
 			boundingBox = {},
 			controls = [],
 			geolocation = true,
 			position = {},
-			tileURI = '//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+			tileURI = '',
```

Comment:
I still don't think we actually need to declare this value here at all because it isn't used anywhere in this file.


`MapOpenStreetMap.es.js` extends this class, but it already defines this value itself, so there is no need to define it in this class here.